clients/src/main/java/org/apache/kafka/common/metrics/QuotaViolationException.java

```diff
@@ -17,15 +17,26 @@
 package org.apache.kafka.common.metrics;
 
 import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.MetricName;
 
 /**
  * Thrown when a sensor records a value that causes a metric to go outside the bounds configured as its quota
  */
 public class QuotaViolationException extends KafkaException {
 
     private static final long serialVersionUID = 1L;
+    public MetricName metricName;
+    public Double value;
+    public Double bound;
```

Comment:
I wouldn't typically add accessors here, although I would have some years back. If you're super keen on this kind of stuff I'll change.